### DISCUSSION TITLE
[BUG] Create Teams issue with New-PnPSite / New-PnPTeamsTeam

### DISCUSSION THREAD

**Discussion by manufz** [pnp/powershell#3141] [General]
### Description
When creating a Team site and adding Teams to it the Team is stuck in setting up state and profile picture can't be set (among other things like getting channel email address). Unfortunately this is still the only way of setting up Teams where the SharePoint default language can be configured.

### Expected behavior 
Teams should eventually complete setup process.

### Actual behavior 
![image](https://github.com/pnp/powershell/assets/30493494/ccb5e4f3-4c4f-48d3-9aec-c400a2faf3a4)

### Steps to reproduce behavior
```
# create team site
$site = New-PnPSite -Type TeamSite -Lcid 1033 -Title "Test" -Description "test" -Owners $owner -Alias "test" -IsPublic:false

#wait for group to be created and get id
$group = Get-PnPAzureADGroup -Identity "test"

# create teams
New-PnPTeamsTeam -GroupId $group.Id
```
Now go to the Teams team > Manage settings > Settings > Change picture. It shows the error message "We're still setting up your team, you can change your picture when we're done."

### What is the version of the Cmdlet module you are running?
2.1.1

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows

**Comment by gautamdsheth** [comment]
Seems like a tenant specific or team specific issue, can't reproduce this at my end.

Can you try adding wait parameter at team creation:

```
$site = New-PnPSite -Type TeamSite -Lcid 1033 -Title "Test" -Description "test" -Owners $owner -Alias "test" -IsPublic:$false -Wait
```

Maybe that can help ? 
Am moving this to a discussion since the cmdlets have no issue as such but the UI seems to be lagging a bit.

Maybe other community members can also chime in , in case they also faced a similar issue.
